src/productcatalogservice/server.go

```diff
@@ -143,6 +144,25 @@ func initStats(exporter *stackdriver.Exporter) {
 	}
 }
 
+// sets the sleepSeconds variable, if env variable provided
+func initSleep() {
+	s := ""
+	if s = os.Getenv("EXTRA_LATENCY"); s == "" {
```

Comment:
You can do all the following in a single if in main(). Don’t introduce this initSleep method. 

you can use if s := syntax also not to do s:=“”.